src/main/java/edu/iis/powp/adapter/MyAdapter.java

```diff
@@ -9,6 +10,8 @@
  * Plotter adapter to drawer with several bugs.
  */
 public class MyAdapter extends DrawPanelController implements IPlotter {
+	
+	DrawPanelController drawerController = DrawerFeature.getDrawerController();

```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.
This field should be private.